app/controllers/api/v1/admin/admin_api_controller.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Admin
+      class AdminApiController < Api::V1::ApiController
+        before_action :authenticate_admin!
```

Comment:
It'd be good to `authenticate_user!` too

Reply:
en authenticate_admin ya se hace el authenticate_user